File: datasets/python/security/eval_usage.py
def execute_code(code):
    """
    Execute arbitrary code - security risk!
    """
    result = eval(code)  # 安全漏洞：使用eval执行任意代码
    return result

if __name__ == "__main__":
    user_input = input("Enter code to execute: ")
    result = execute_code(user_input)
    print(f"Result: {result}")

File: datasets/javascript/security/eval_usage.js
/**
 * Execute arbitrary code - security risk!
 */
function executeCode(code) {
    const result = eval(code); // 安全漏洞：使用eval执行任意代码
    return result;
}

if (typeof module !== 'undefined' && module.exports) {
    module.exports = executeCode;
} else {
    window.executeCode = executeCode;
}

// Test
const userInput = prompt("Enter code to execute: ");
const result = executeCode(userInput);
console.log(`Result: ${result}`);